framework/doc/content/source/mfem/equation_systems/TimeDependentEquationSystem.md

```diff
@@ -0,0 +1,28 @@
+# TimeDependentEquationSystem
+
+!if! function=hasCapability('mfem')
+
+For transient problems, time derivatives $\dot{u}$ of the trial variables $u$ will also be
+present in the weak form.
+
+!equation
+\left(\mathcal{T}(\dot{u}), v\right)_{\Omega} + {\left(\mathcal{L}(u), v\right)_{\Omega}
+=\left(f,v\right)_{\Omega}\,\,\,\forall v \in V}
+
+Contributions to $\left(\mathcal{T}(\phi_j), \varphi_i\right)_{\Omega}$ are given by
+time derivative kernels such as
+ [MFEMTimeDerivativeMassKernel](source/mfem/kernels/MFEMTimeDerivativeMassKernel.md)
+
+Currently, transient problems in MOOSE's wrapping of MFEM are solved using an implicit backwards
+Euler method.
```

Comment:
Do we have an issue/is there a plan for different time integration schemes? We have someone within the NEAMS program who has talked about leveraging or implementing (I can't remember which) exponential time integrators in MFEM

Reply:
Nice. Let's make a MOOSE issue for this soon if it doesn't already exist